packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -115,13 +116,15 @@ export default function SidebarNavigationScreenGlobalStyles() {
 		},
 		[]
 	);
+	const { set: setPreference } = useDispatch( preferencesStore );
 
 	const openGlobalStyles = useCallback( async () => {
 		return Promise.all( [
+			setPreference( 'core', 'distractionFree', false ),
```

Comment:
set DFM to false when editing global styles here instead of the open general sidebar action